docs/requirements.md

```diff
@@ -1,51 +1,140 @@
+- Use `batch` or `group` for `commitlog_sync`.
+- (In case you use Cassandra-compatible databases) Use Cassandra-compatible databases that properly support Lightweight Transactions (LWT).
 
-## JDBC database recommendations
+#### Analytics
+{:.no_toc}
+- N/A (Thee are no Cassandra-specific requirements since Cassandra always returns committed records.)
 
-In ScalarDB on JDBC databases, you can't choose a consistency level (`LINEARIZABLE`, `SEQUENTIAL` or `EVENTUAL`) in your code by using the `Operation.withConsistency()` method. In addition, the consistency level depends on the setup of your JDBC database.
+</div>
+</div>
 
-For example, if you have asynchronous read replicas in your setup and perform read operations against them, the consistency will be eventual because you can read stale data from the read replicas. On the other hand, if you perform all operations against a single master instance, the consistency will be linearizable.
+## Recommendations
 
-With this in mind, you must perform all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. In other words, ScalarDB does not support read replicas. 
+ScalarDB recommends each underlying database to be configured properly for high performance and high availability. The following items are some knobs and configurations to update.
 
 {% capture notice--info %}
 **Note**
 
-You can still use a read replica as a backup and standby even when following this guideline.
+ScalarDB can be seen as an application of underlying databases, so it is worth trying other knobs and configurations that are commonly effective.
 {% endcapture %}
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+<div id="tabset-2">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'JDBC_databases2', 'tabset-2')" id="defaultOpen-2">JDBC databases</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB2', 'tabset-2')">DynamoDB</button>
+  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL2', 'tabset-2')">Cosmos DB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'Cassandra2', 'tabset-2')">Cassandra</button>
+</div>
+
+<div id="JDBC_databases2" class="tabcontent" markdown="1">
+- Use read committed isolation for better performance.
+- Follow each database’s best practices for performance optimization. For example, increasing buffer size (e.g., shared_buffers in PostgreSQL) and increasing the number of connections (e.g., max_connections in PostgreSQL) are usually recommended for better performance.
+</div>
 
+<div id="DynamoDB2" class="tabcontent" markdown="1">
+- Increase RCU and WCU for high throughput.
+- Enable Point-in-time Recovery (PITR).
+
+{% capture notice--info %}
+**Note**
+
+Since DynamoDB stores data in multiple availability zones by default, so you don’t need to do anything for better availability.)
```

Comment:
I wanted to be specific here, so I've re-worded `do anything`. I'm open to using other, more specific phrases though.

```suggestion
Since DynamoDB stores data in multiple availability zones by default, you don’t need to adjust any configurations to improve availability.
```